src/js/get-fundraiser.js

```diff
@@ -1,20 +1,102 @@
 const mapping = {
-  // 14274: { // France
+  // France
+  // 14274: { // FFME - Rockclimber
   //   url: ' https://www.helloasso.com/associations/rockclimber/formulaires/1/widget'
   // },
+  
+  //Greenspit
```

Comment:
please add space after `//`, it eases the reading

Reply:
OK, I'll do.

We should find a way to transmit to the children ...